src/components/activity-page/iframe-interactive/iframe-interactive.tsx

```diff
@@ -9,11 +9,12 @@ interface IProps {
 
 export class IframeInteractiveQuestion extends React.PureComponent<IProps>  {
   render () {
-    const { embeddable, questionNumber } =this.props;
+    const { embeddable, questionNumber } = this.props;
+    const questionName = embeddable.name ? `: ${embeddable.name}` : "";
     return(
-      <div className="question" data-cy="iframe-interactive-question">
-        <div className="header">Question #{questionNumber}</div>
-        <iframe className="content" src={embeddable.url}></iframe>
+      <div className="iframe-interactive" data-cy="iframe-interactive-question">
+        { questionNumber && <div className="iframe-interactive-header">Question #{questionNumber}{questionName}</div> }
+        <iframe className={`iframe-interactive-content ${questionNumber ? "vertical-offset" : ""}`} src={embeddable.url}></iframe>
```

Comment:
Don't plain iframe interactives (MwInteractives) also support authored and student state? Couldn't we have `IframeRuntime` here? Or is that just not in the scope of this PR?

Reply:
Scott mentioned this as well.  I wasn't really thinking about it in this PR, but it does make sense.  I think I will try to make that change and use `IFrameRuntime`.